src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -58,20 +58,28 @@
 ;;; their implicit defaults reflect the security issue.  The latter is far more
 ;;; risky for our downstream apps, thus it was decided that it makes sense to
 ;;; keep these overrides.
-(def acceptable-ciphers
-  ["TLS_DHE_RSA_WITH_AES_128_GCM_SHA256"
-   "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"
-   "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
-   "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384"
+(def tls-1-dot-3-ciphers
+  ["TLS_AES_128_GCM_SHA256"
+   "TLS_AES_256_GCM_SHA384"
+   "TLS_CHACHA20_POLY1305_SHA256"])
```

Comment:
what'd Shellee say about these three vs the five listed in the ticket?

Reply:
she hasn't answered yet